src/thrift/parquet.thrift

```diff
@@ -84,12 +84,12 @@ enum ConvertedType {
    * Stored as days since Unix epoch, encoded as the INT32 physical type.
    *
    */
-  DATE = 6; 
```

Comment:
Could you remove all of the whitespace changes? Since this is the file format definition, it's more important to keep modifications to a minimum so that we can trace back changes through version control.